### ISSUE TITLE
about tf.train.ExponentialMovingAverage , whats the use in argman'EAST？

### ISSUE THREAD

**Issue by fengxin619** [kurapan/EAST#48]
in keras,  i dont find the part ?    tf.train.ExponentialMovingAverage  .....dose this affect the mode's effect？

**Comment by kurapan** [comment]
Yes, it doesn't use moving average. It may slightly hurt the performance but the difference shouldn't be very big.
